numpy-stubs/__init__.pyi

```diff
@@ -309,6 +309,8 @@ class ndarray(Generic[_DType]):
     def __add__(self, value: ndarray[_DType]) -> ndarray[_DType]: ...
     @overload
     def __add__(self, value: _DType) -> ndarray[_DType]: ...
+    @overload
+    def __add__(self, value: _FloatLike) -> ndarray[_DType]: ...
```

Comment:
I would prefer this:
```suggestion
    def __add__(self, value: float) -> ndarray[float64]: ...
```
because the other things in `_FloatLike` should already be covered by `_DType`